nix/status-go/source.nix

```diff
@@ -14,6 +14,7 @@ let
     rev = "unknown";
     shortRev = rev;
     rawVersion = "develop";
+    version = rawVersion;
```

Comment:
I second what @jakubgs said in https://github.com/status-im/status-mobile/pull/22041#pullrequestreview-2608236770.

Side question: I see `rawVersion` being set below, but I couldn't find any usage of it. Is this value used by some other tool? Could we remove it? https://github.com/status-im/status-mobile/blob/ed10ed70f756256a4ca23da123fec5414c3e3b38/nix/status-go/source.nix#L46

Reply:
Hmm not sure if the `rawVersion` field is needed. I don't see it used anywhere in our nix files, but perhaps it's a convention (?).

Reply:
yeah good point! I do not see it there either, probably something we've done and can probably remove.

@siddarthkay do you have an opinion on keeping `rawVersion`?

Reply:
I think it's just there for me to be able to inspect how it was before being handled by `lib.sanitizeVersion`. Safe to remove.